web/src/app/users/UserContactMethodVerificationForm.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react'
+import p from 'prop-types'
+import Grid from '@material-ui/core/Grid'
+import TextField from '@material-ui/core/TextField'
+import LoadingButton from '../loading/components/LoadingButton'
+import { Mutation } from 'react-apollo'
+import { makeStyles } from '@material-ui/core/styles'
+import { graphql2Client } from '../apollo'
+import { FormContainer, FormField } from '../forms'
+import { sendVerificationCodeMutation } from './UserContactMethodVerificationDialog'
+
+const useStyles = makeStyles({
+  fieldGridItem: {
+    flexGrow: 1,
+  },
+  sendGridItem: {
+    display: 'flex',
+    alignItems: 'center',
+  },
+})
+
+export default function UserContactMethodVerificationForm(props) {
+  const classes = useStyles()
+
+  function getTitle() {
```

Comment:
Is this needed now? Since with the new design, clicking on either the warning icon or Reactivate button does the action of sending the code.
Is there a scenario where we could get into a state with the dialog open but code not sent yet?

Reply:
I did want to keep Send/Resend in just in the case that if the send fails on the initial dialog opening, it would show the error message and an option to try sending it manually. From there it would switch to resend (once the message is actually sent at least once). If we feel this isn't necessary I can take it out and always have the button read as "Resend"

Reply:
Wouldn't that just always flicker then? (if when opening the dialog we always send it first), also the mutation returns once the send is queued but not when it's actually sent -- we don't yet have that info (or send errors) on the UI.

Reply:
So do we want to just default to "Resend" then, and do away with the `sendAttempted` props?